python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -130,3 +173,17 @@ def prepare_vpdq_feature(
     """
     features = json_to_vpdq(signal_str)
     return dedupe(quality_filter(features, quality_tolerance))
+
+
+def go():
```

Comment:
Did you mean to check this function in? I don't see usages of go() in this diff.

Reply:
Derp, no. Thanks for the catch. It's easier to paste into python if you you type it into vscode first, because shift+enter will execute it.